src/main/java/org/veupathdb/lib/container/jaxrs/repo/SQL.java

```diff
@@ -3,7 +3,10 @@
 import java.util.function.Supplier;
 
 import io.vulpine.lib.sql.load.SqlLoader;
+import org.veupathdb.lib.container.jaxrs.providers.OptionsProvider;
 
```

Comment:
Added the TODO rather than doing the task here because it wasn't in scope for the PR and I didn't want to muddy the waters